### DISCUSSION TITLE
Unlock issue

### DISCUSSION THREAD

**Discussion by amaioli** [ctubio/Krypto-trading-bot#1161] [General]
Hi,

I'm very happy to Krypto bot feature. I've tried to unlock it sending the btc to the address advised on the dashboard but the bot is still locked. I've tried to restart it, the bot is running on docker.

Anyone can help me?

Thanks in advance

**Comment by ctubio** [comment]
:koala: the GATEIO bot was unlocked some minutes ago, can you restart it?
